tests/ci/cdk/cdk/codebuild/github_ci_analytics_omnibus.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+# Doc for batch https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list
+batch:
+  build-list:
+
+    - identifier: ubuntu2004_clang10_x86_64_analytics
+      buildspec: ./tests/ci/codebuild/common/run_analytics.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        image: X86_ECR_REPO_PLACEHOLDER:ubuntu-20.04_clang-10x_latest
+
+    - identifier: ubuntu2004_clang10_arm_analytics
+      buildspec: ./tests/ci/codebuild/common/run_analytics.yml
+      env:
+        type: ARM_CONTAINER
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
```

Comment:
Nit: arm `BUILD_GENERAL1_SMALL` can work for metrics publish?
https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-compute-types.html